app/presenters/idv/by_mail/letter_enqueued_presenter.rb

```diff
@@ -24,21 +24,25 @@ def address_lines
       end
 
       def button_text
-        if sp
-          t('idv.cancel.actions.exit', app_name: APP_NAME)
-        else
-          t('idv.buttons.continue_plain')
-        end
+        t('idv.cancel.actions.exit', app_name: APP_NAME)
```

Comment:
If this is a straightforward translation can we move it out of the presenter and into the view?

Reply:
I was on the fence. The usage was:

```erb
<%= link_to(
      @presenter.button_text,
      @presenter.button_destination,
 ...
```

so I had initially figured I'd leave it. I don't feel strongly at all, though. Happy to just inline this. 👍